### ISSUE TITLE
Still shows border when embedded

### ISSUE THREAD

**Issue by igorsantos07** [artem-sedykh/mini-climate-card#122]
I'm not sure when this started, but I guess it was together with the Lovelace border/radius changes last year. I never reported because, well, I thought it would be noticed and fixed, but it seems #106 didn't fix it. I'm already on v2.6.0.

![image](https://github.com/artem-sedykh/mini-climate-card/assets/532299/6e07a617-f097-4698-9068-473e83cf19c5) 

As far as I can see, my dashboard's theme is `Backend-selected` (which defaults to HA's dark theme), and the card above has no custom theming either.
On the inspector, the border seems to come from `div.mc__bg` (1.25px on each side) when in the default theme.

**Comment by igorsantos07** [comment]
lol I didn't recall that setting, but you know what? It was _already_ present.

![image](https://github.com/artem-sedykh/mini-climate-card/assets/532299/df1f62e0-d223-4042-84c0-138a12db2ab1)

**Comment by regevbr** [comment]
So what is causing the shadow render?

**Comment by igorsantos07** [comment]
That's exactly why I opened a bug report :))

From the screenshot you sent, you can see this is already active because the up arrow is almost crossing the top border (what's caused by the lack of padding). I've just set it to `false` and I see the difference, there's a lot more spacing and extra shadows, indeed.

I think we should noticed that this `group` option doesn't seem to touch on borders, which is what's going on and I mentioned in the original report - not shadows, as you stated. Other than saying this, I don't know much else to fiddle / search :sweat_smile: 

Should `group: true` disable shadows on `div.mc__bg`?

**Comment by regevbr** [comment]
Please provide which css class causes it

**Comment by igorsantos07** [comment]
>On the inspector, the border seems to come from `div.mc__bg` (1.25px on each side) when in the default theme.

**Comment by regevbr** [comment]
what css property that you will manually put will disable it?

**Comment by regevbr** [comment]
fixed in #123 and released ini 2.6.1 can you please confirm?

**Comment by igorsantos07** [comment]
![image](https://github.com/artem-sedykh/mini-climate-card/assets/532299/083a4bb1-a914-4331-9494-20bfc790aebb)
As pretty as it could get. Thank you so much, @regevbr!

**Comment by regevbr** [comment]
screenshot please

**Comment by regevbr** [comment]
For me it works... Top one has group set to true, bottom one has group set to false
![image](https://github.com/artem-sedykh/mini-climate-card/assets/5775519/eed82d23-d795-4c27-ba12-30fca5c24b52)

**Comment by theOrakle** [comment]
Well then I must have something going on...  Thanks for looking at it and sorry to be necromancer.

**Comment by regevbr** [comment]
I would love to help if you can figure out which CSS property is causing it, something like that:

![image](https://github.com/artem-sedykh/mini-climate-card/assets/5775519/47e1c06b-e4da-4732-a77f-f298d40dc926)

**Comment by jossiee** [comment]
looks like i got this problem... has this ever been fixed? 

in the inspect screen it still appears to come from `mc__bg`
![image](https://github.com/artem-sedykh/mini-climate-card/assets/8819339/b643e584-a16d-4819-aa26-4cfd0b757815)

```
type: custom:stack-in-card
title: Slaapkamer achter
cards:
  - type: custom:mini-climate
    entity: climate.slaapkamer_achter
    icon: mdi:thermostat
    group: true
    hide_current_temperature: true
    hvac_mode:
      hide: true
    fan_mode:
      hide: true
    secondary_info:
      type: hvac-action
      source:
        heating:
          icon: mdi:radiator
          name: ''
        idle:
          icon: mdi:radiator-off
          name: ''
    indicators:
      humidity:
        icon: mdi:water
        source:
          entity: sensor.slaapkamer_achter_rv
        unit: '%'
      temperature:
        icon: mdi:thermometer
        source:
          entity: sensor.slaapkamer_achter_temp
        unit: °C
    card-mod:
      style:
        ha-card: |
          div {
            box-shadow: none !important;
          }
  - type: custom:mini-graph-card
    animate: true
    entities:
      - entity: climate.slaapkamer_achter
        show_fill: false
        color: '#0000ff'
        show_state: false
        attribute: temperature
      - entity: climate.slaapkamer_achter
        show_fill: true
        color: '#00ff00'
        show_state: false
        attribute: current_temperature
      - color: gray
        entity: sensor.omdag_aan_uit
        name: Night
        show_line: false
        show_points: false
        show_legend: false
        y_axis: secondary
    points_per_hour: 4
    hours_to_show: 168
    line_width: 2
    decimals: 0
    upper_bound_secondary: 0.9
    lower_bound_secondary: 0.1
    show:
      icon: false
      name: false
      points: false
      labels: true
      state: false
      labels_secondary: false
      legend: false
```

This is my configuration

and this is what is looks like.
![image](https://github.com/artem-sedykh/mini-climate-card/assets/8819339/6128b2d5-d6f3-4636-b7e5-8ec964274701)

Please let me know if there is anything i can check or test!

**Comment by regevbr** [comment]
it seems that your theme configures the `ha-card-box-shadow` variable, which causes it. It is the same shadow applied to the main card in the picture you sent.
I will create a PR later that allows you to define a new variable that will overide it. If you change manually the value to `none` it solves it for you?

**Comment by jossiee** [comment]
Yes if i edit in the inspect screen it is gone

Thanks for the quick reply!

Op do 22 feb 2024 om 16:15 schreef Regev Brody ***@***.***>

**Comment by regevbr** [comment]
released in 2.7.1
See https://github.com/artem-sedykh/mini-climate-card/pull/148/files for the relevant changes
